documentation/book/con-cluster-operator-watch-options.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// assembly-operators-cluster-operator.adoc
+// assembly-cluster-operator.adoc
+
+[id='con-cluster-operator-watch-options-{context}']
+
+= Watch options for a Cluster Operator deployment
+
+When the Cluster Operator is running, it starts to _watch_ Kafka resources for updates.
+
+Depending on the deployment, the Cluster Operator can _watch_ Kafka resources from:
+
+* xref:deploying-cluster-operator-{context}[A single namespace (the namespace it is installed)]
+* xref:deploying-cluster-operator-to-watch-multiple-namespaces{context}[Multiple namespaces]
+* xref:deploying-cluster-operator-to-watch-whole-cluster-{context}[All namespaces]
+
+NOTE: {ProductName} provides example YAML files to make the deployment process easier.
+
+Cluster Operator watches the following resources:
+
+* A `Kafka` resource for the Kafka cluster.
+* A `KafkaConnect` resource for the Kafka Connect cluster.
+* A `KafkaConnectS2I` resource for the Kafka Connect cluster with Source2Image support.
+* A `KafkaMirrorMaker` resource for the Kafka Mirror Maker instance.
+
+When a new `Kafka`, `KafkaConnect`, `KafkaConnectS2I`, or `Kafka Mirror Maker` resource is created in the Kubernetes cluster, the operator gets the cluster description from the resource and starts creating a new Kafka, Kafka Connect, or Kafka Mirror Maker cluster by creating the necessary Kubernetes resources, such as StatefulSets, Services and ConfigMaps.
```

Comment:
Remove spaces: `Kafka Mirror Maker` resource is created

Reply:
This section was rewritten, so change not required.